mobile/status.go

```diff
@@ -2323,6 +2323,58 @@ func addCentralizedMetric(requestJSON string) string {
 	return metric.ID
 }
 
+func SetLogLevel(requestJSON string) string {
```

Comment:
Should we deprecate the functions `ext/api.go` ? At least to me it's confusing to see duplicated public APIs for `SetLogLevel`, `SetLogEnabled`, and `SetLogNamespaces`. If not to deprecate, then could you elaborate on the purpose of the functions in `ext/api.go`?